src/index.js

```diff
@@ -19,7 +19,7 @@ const schema = require('./options.json');
 
 const defaults = {
   cacheContext: '',
-  cacheDirectory: path.resolve('.cache-loader'),
+  cacheDirectory: path.resolve('./node_modules/.cache/.cache-loader'),
```

Comment:
Can be broken when you use lerna and other nonstandard project structure, to avoid this we need use `find-cache-dir` package

Reply:
@evilebottnawi you're right, just forgot about this! Thanks for the feedback 👍